static/app/views/issueList/overview.tsx

```diff
@@ -660,7 +660,9 @@ class IssueListOverview extends React.Component<Props, State> {
 
   onSelectStatsPeriod = (period: string) => {
     if (period !== this.getGroupStatsPeriod()) {
-      this.transitionTo({groupStatsPeriod: period});
+      const cursor = this.props.location.query.cursor;
+      const queryPageInt = parseInt(this.props.location.query.page, 10);
```

Comment:
Would it be worth to guard from NaN here in case we end up with a malformed query.page value? I'm not sure if it's possible (I'm lacking context), but could a navigation from an entirely different page with a period param (and possibly malformed or absent page param) trigger this?

I'm approving bc my comment is more of a question and dont want to hold up merging :)

Reply:
pushed an update to guard against this!